### DISCUSSION TITLE
cpu core assignment possible :) ?

### DISCUSSION THREAD

**Discussion by u-an-i** [albar965/littlenavmap#668] [Ideas]
Ms FS (2020) for example utilises only 6 cores I read a while back.

Running LNM on the last of availables cores might result in better performance for LNM And Ms FS (2020) (if it matters).

**Comment by albar965** [comment]
There are third party tools for this like Process lasso or Sysinternals tools.
Normally I'd leave it up to the OS to care about CPU affinity.

**Reply by u-an-i** [reply]
thank you for the pointers. I will see what i can achieve!

I did have the impression running LNM + webserver + serving 2 clients at 1s refresh rate does impact Ms FS 2020 but i am not sure.
